device/celestica/x86_64-cel_silverstone-r0/plugins/sfputil.py

```diff
@@ -184,10 +275,14 @@ class SfpUtil(SfpUtilBase):
     SFP_PORT_START = 33
     SFP_PORT_END = 34
 
+    NUM_OSFP = 32
+
     EEPROM_OFFSET = 9
     PORT_INFO_PATH = '/sys/class/silverstone_fpga'
     QSFP_DD_DOM_OFFSET = 2304
 
+    POLL_INTERVAL = 1
```

Comment:
Is this seconds or miliseconds,  please make a comment.

Reply:
updated